src/main/java/com/extremelyd1/gameboard/GameBoard.java

```diff
@@ -61,7 +61,19 @@ public GameBoard(Game game) {
     public void update() {
         // Clear all existing scores
```

Comment:
This comment is not relevant anymore, perhaps document the update logic a bit more. While I can understand it, it might not be as clear for people unfamiliar with the codebase.

Reply:
Updated the java docs to explain the process and updated this comment